tests/unit/charms/tls_certificates_interface/v4/test_tls_certificates_v4.py

```diff
@@ -369,6 +369,24 @@ def test_given_certificate_signin_request_when_from_csr_then_attributes_are_corr
     assert attributes.locality_name == "Montreal"
 
 
+def test_generated_ca_does_not_have_an_empty_sans_extension():
```

Comment:
This test name should conform to the convention `test_given_X_when_Y_then_Z`, maybe that will make it more clear what we're testing here.